userdocs/diagnostics/statement-expression-out-of-place.md

```diff
@@ -0,0 +1,45 @@
+# Statement may only be used as an expression in return, throw, or as the source of an assignment
+
+Using an `if` or `switch` statement as an expression is only valid in certain contexts.
```

Comment:
Same comment as above about deliberately writing a summary line before starting the article.